File: C Language/Q25.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b, c, s, area;

    printf("Enter the three sides of the triangle:\n");
    scanf("%lf %lf %lf", &a, &b, &c);

    // Calculate semi-perimeter
    s = (a + b + c) / 2;

    // Heron's formula
    area = sqrt(s * (s - a) * (s - b) * (s - c));

    printf("The area of the triangle is: %.2f\n", area);

    return 0;
}

File: Python Language/Q25.py
# Program to find area of a triangle given three sides

import math

# Input sides
a = float(input("Enter side a: "))
b = float(input("Enter side b: "))
c = float(input("Enter side c: "))

# Calculate semi-perimeter 's'
s = (a + b + c) / 2

# Calculate Area using Heron's Formula: sqrt(s * (s-a) * (s-b) * (s-c))
area = math.sqrt(s * (s - a) * (s - b) * (s - c))

print(f"The area of the triangle is: {area:.2f}")